main.py

```diff
@@ -5,12 +5,12 @@
 import os
 
 
+# This function is responsible for checking the files names in specific folder if they starts with "deep".
 def get_files():
+    # getting the folder path from the user.

```

Comment:
Please read a turtorial about how to document in python and try again.
It should include data about the parameters it gets and about the return value.